src/components/pages/devices/flyouts/deviceDetails/propertyGrid/propertyGrid.scss

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/mixins';
+@import 'src/styles/themes';
+
+.grid-container {
```

Comment:
`grid-container` needs to be renamed `property-grid-container`. If this is checked in it will result in a name collision with the grid used in the dashboard